docs/tutorial.md

```diff
@@ -4,10 +4,23 @@ title: Step-by-Step Tutorial
 description: This document provides a step-by-step tutorial for the reSolve framework. Throughout this tutorial, you will iteratively develop a ShoppingList application and learn fundamental concepts of the reSolve framework.
 ---
 
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
 This document provides a step-by-step tutorial for the reSolve framework.
 Throughout this tutorial, you will iteratively develop a ShoppingList application and learn fundamental concepts of the reSolve framework.
```

Comment:
What do you mean by "iteratively" in this context? Are you repeating procedures or steps over and over?

Reply:
It seems, I have chosen a completely wrong word. What I was trying to say is "gradually and in small adjustments". But we already stated that the tutorial is "step-by-step", which means the same, so I think I should just remove the word.

Reply:
Thank you. I've added the suggestion below:

```suggestion
Throughout this tutorial, you will develop a ShoppingList application and learn fundamental concepts of the reSolve framework.
```